apps/meteor/tests/e2e/message-actions.spec.ts

```diff
@@ -137,15 +137,20 @@ test.describe.serial('message-actions', () => {
 
 	test('expect create a discussion from message', async ({ page }) => {
 		const message = `Message for discussion - ${Date.now()}`;
+		const discussionName = `Discussion Name - ${Date.now()}`;
 
 		await poHomeChannel.content.sendMessage(message);
 		await poHomeChannel.content.openLastMessageMenu();
 		await page.locator('role=menuitem[name="Start a Discussion"]').click();
 		const createButton = page.getByRole('dialog').getByRole('button', { name: 'create' });
 		// Name should be prefilled thus making the create button enabled
 		await expect(createButton).not.toBeDisabled();
+		await page.locator('role=textbox[name="Name"]').fill(discussionName);
 		await createButton.click();
-		await expect(page.locator('header h1')).toHaveText(message);
+		await expect(page.locator('header h1')).toHaveText(discussionName);
+		await poHomeChannel.sidenav.openChat(targetChannel);
+		// Should fail if more than one discussion has been created
+		await expect(page.locator('div.messages-box').getByText(discussionName)).toBeVisible();
```

Comment:
let's try to use a `pageObj` from home-content, I'm pretty sure there are something you could use there